backend/apps/billing/serializers.py

```
@@ -0,0 +1,20 @@
+class BountySerializer(serializers.ModelSerializer):
+    sponsor_name = serializers.CharField(source='sponsor.name', read_only=True)
+    
+    class Meta:
+        model = Bounty
+        fields = ['id', 'sponsor', 'sponsor_name', 'title', 'description', 'amount', 'currency', 'is_active', 'created_at']
+
+class BountyClaimSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = BountyClaim
+        fields = '__all__'
+        read_only_fields = ['payout_id', 'is_approved']
```

Ruff (0.15.21)

[warning] Mutable default value for class attribute

(RUF012)